bee-api/bee-rest-api/src/endpoints/routes/api/v1/messages/mod.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    types::{
+        body::{DefaultErrorResponse, ErrorBody, SuccessBody},
+        dtos::MessageDto,
+        responses::{MessageChildrenResponse, MessageMetadataResponse, MessageResponse, SubmitMessageResponse},
+    },
+    AppStorage,
+};
+
+use bee_message::{Message, MessageId};
+use bee_storage::access::{Fetch, Insert};
+
+use axum::{
+    extract::{Extension, Path},
+    response::Json,
+    routing::{get, post},
+    Router,
+};
+use serde_json::{json, Value};
+
+use rand::Rng;
+use std::{convert::TryFrom, str::FromStr, sync::Arc};
+
+pub fn api_routes() -> Router {
+    Router::new()
+        .route("/", post(post_handler))
+        .route("/:messageId", get(get_id_handler))
+        .route("/:messageId/metadata", get(get_id_metadata_handler))
+        .route("/:messageId/raw", get(get_id_raw_handler))
+        .route("/:messageId/children", get(get_id_children_handler))
+}
+
+pub async fn post_handler(
+    Extension(app_storage): Extension<Arc<AppStorage>>,
+    Json(payload): Json<MessageDto>,
+) -> Json<Value> {
+    let mut rng = rand::thread_rng();
+
+    let rand_part1: u128 = rng.gen();
+    let rand_part2: u128 = rng.gen();
+    let rand_id = format!("{:x}{:x}", rand_part1, rand_part2);
```

Comment:
Just for understanding: is this a "fake/dummy implementation"? Asking because of the random message id generation.

Furthermore: this endpoint allows the user to post a message to the node, right?
So if thats the case we can't just put the message into the storage, instead this endpoint needs to forward incoming message to the "node message worker" that proccesses messages. I think there is however no "node message worker" yet. Do you agree @thibault-martinez ?

Reply:
Yes, it's just a dummy implementation without any workers.